module/VuFind/src/VuFind/AjaxHandler/GetSideFacets.php

```diff
@@ -241,11 +241,9 @@ protected function formatFacets(
         $hierarchicalFacets = [];
         $hierarchicalFacetSortOptions = [];
         $options = $results->getOptions();
-        if (is_callable([$options, 'getHierarchicalFacets'])) {
-            $hierarchicalFacets = $options->getHierarchicalFacets();
-            $hierarchicalFacetSortOptions
-                = $recommend->getHierarchicalFacetSortOptions();
-        }
+        $hierarchicalFacets = $options->getHierarchicalFacets();
```

Comment:
You can now eliminate the empty array initializations of these variables above.

Reply:
Oops, done.